spec/features/bulk_jobs_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'spec_helper'
+
+# Feature tests for the spreadsheet bulk uploads view.
+feature 'Bulk jobs view', js: true do
+
+  before :each do
+    @current_user = double(
+      :webauth_user,
+      login: 'sunetid',
+      logged_in?: true,
+      permitted_apos: [],
+      is_admin: true
+    )
+    allow(@current_user).to receive(:roles).and_return([])  # necessary to determine if current user has right permissions
+    allow_any_instance_of(ApplicationController).to receive(:current_user).
+                                                    and_return(@current_user)
+  end
+
+  scenario 'the submit button exists' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.